lab_5_scrapper/target_score.txt

```diff
@@ -1 +1 @@
-6
\ No newline at end of file
+10
```

Comment:
do you plan to increase your mark?

Reply:
yes, i've changed the score to this moment to be able to work with lab 6 but I still plan to raise my grade for the 5 lab

Reply:
ok, let's first handle pipeline, then you will try to increase